src/main.js

```diff
@@ -1,5 +1,9 @@
 // query selector variables go here 👇
 
+var image = document.querySelector('.poster-img');
+var title = document.querySelector('.poster-title');
+var quote = document.querySelector('.poster-quote');
```

Comment:
Nice - the variables appear to be mapped to the correct html elements by class name!